app/src/main/event/controllers/EventCtrl.js

```diff
@@ -95,10 +95,24 @@ angular.module('Xenia.Event')
             return event.prizesQueue.length > 1 && event.winner.id > 0;
         };
 
-        event.drawCandidate = function(giveawayId) {
+        event.drawCandidateAndSkipCurrentWinner = function(giveawayId) {
+            var skippedMemberId = (typeof event.winner.id != "undefined") ? event.winner.id : 0;
```

Comment:
a to nie przejdzie po prostu jako `!event.winner.id ? event.winner.id : 0`? albo nawet `event.winner.id || 0`?